add meta description (#2184)
FlowFuse/website: src/ask-me-anything/ama-nodered-april.md

---
title: Ask Me Anything about Node-RED (April Edition)
subtitle: Join Nick O'Leary, co-creator of Node-RED & CTO at FlowFuse and Rob Marcer, Developer Educator at FlowFuse, for an AMA on Node-RED

> description: Join Nick O'Leary and Rob Marcer for an Ask Me Anything on Node-RED! You ask, we answer. Get expert solutions now!

Reviewer: ```suggestion
description: Nick O'Leary and Rob Marcer answer your questions on Node-RED
```
